spec/calculator.spec.js

```diff
@@ -1,24 +1,31 @@
-'use strict';
+(function (factory) {
+  if (typeof require === 'function' && typeof exports === 'object' && typeof module === 'object') {
+    module.exports = factory(require('chai'), require('../Calculator'));
+  } else {
+    factory(chai, Calculator);
```

Comment:
This code will run if the test is being run in the browser. It does not need to use `require(...)` because chai and Calculator are already inside the global scope. It can simply pass them in to our tests via the factory function. One more thing to notice is that our factory function does not return anything, so we don't need to do anything with the return value.